bootstrap.sh

```diff
@@ -17,7 +17,11 @@ export APP_ROOT=$(pwd)
 export WORKSPACE=${WORKSPACE:-$APP_ROOT}  # if running in jenkins, use the build's workspace
 export BONFIRE_ROOT=${WORKSPACE}/.bonfire
 export CICD_ROOT=${BONFIRE_ROOT}
-export IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+if [[ ! -n "$IMAGE_TAG" ]]; then
```

Comment:
Right, I have to set it prior to this point. The reason for this is because `export IMAGE_TAG=$(git rev-parse --short=7 HEAD)` results in the tag from the cicd-tools repo not the repository that will be running the pr check.